mitodl/salt-ops
# Add GTM for OCW-Studio apps

#### What are the relevant tickets?

fixes https://github.com/mitodl/salt-ops/issues/1355
checks a box on https://github.com/mitodl/ocw-www/issues/19
checks a box on https://github.com/mitodl/salt-ops/issues/1343

FYI @mbertrand 

#### What's this PR do?

Adds GTM keys for each deployment of OCW Studio

#### How should this be manually tested?

Visit a page that includes the Google Tag code and confirm that an event shows up in the Analytics account (assuming I configured the Google Tag Manager correctly)

#### Any background context you want to provide?

Let me know if you need access to the Google Analytics account or the Google Tag Manager account.
